treeherder/perf/models.py

```diff
@@ -48,6 +48,11 @@ class PerformanceSignature(models.Model):
                                          null=True, blank=True)
     has_subtests = models.BooleanField()
 
+    # we treat suite & test as human unreadable identifiers
+    #  these are their human readable form
+    suite_public_name = models.CharField(max_length=30, null=True)
+    test_public_name = models.CharField(max_length=30, null=True)
```

Comment:
Is this meant to replace use of this: https://github.com/mozilla/treeherder/blob/master/treeherder/webapp/api/performance_serializers.py#L249

Reply:
Thanks for asking. No, it will come from mozilla-central and we won't change it. Actually, it will change the output of PerformanceSummarySerializer.get_name method. The ingestion of suite_public_name/test_public_name will be done in another bug.

Reply:
So it'll change the output but won't replace the need for the composite name that's created in get_name and in the PerfSeriesModel, correct?

Reply:
That's correct.